src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -12,23 +12,23 @@ public static unsafe partial class Path
     {
         private static volatile delegate* unmanaged<int, char*, uint> s_GetTempPathWFunc;
 
-        public static char[] GetInvalidFileNameChars() => new char[]
-        {
+        public static char[] GetInvalidFileNameChars() =>
+        [
             '\"', '<', '>', '|', '\0',
             (char)1, (char)2, (char)3, (char)4, (char)5, (char)6, (char)7, (char)8, (char)9, (char)10,
             (char)11, (char)12, (char)13, (char)14, (char)15, (char)16, (char)17, (char)18, (char)19, (char)20,
             (char)21, (char)22, (char)23, (char)24, (char)25, (char)26, (char)27, (char)28, (char)29, (char)30,
             (char)31, ':', '*', '?', '\\', '/'
-        };
+        ];
 
-        public static char[] GetInvalidPathChars() => new char[]
-        {
+        public static char[] GetInvalidPathChars() =>
```

Comment:
This might be a case where it'd be nice to have a way to get a ROSpan instead, for allocation free handling

Reply:
Yes, we could add a new property/method. Though we ourselves have been moving away from relying on this for validation and instead relying on the OS, so while there are still cases where it's relevant, they're becoming more sparse.